app/battle_i.py

```diff
@@ -0,0 +1,37 @@
+from typing import Dict

```

Comment:
Change the naming of this file to be more descriptive.

Reply:
The file name is not changed, the presence of `_i` is not clear. Please, rename the file.